src/Mod/TechDraw/App/DrawProjGroup.cpp

```diff
@@ -1190,31 +1144,20 @@ void DrawProjGroup::updateSecondaryDirs()
     recomputeChildren();
 }
 
-void DrawProjGroup::rotate(const std::string& rotationdirection)
+void DrawProjGroup::rotate(const RotationMotion& rotationdirection)

```

Comment:
Seems to be an error here as `RotationMotion` is used but it is called like the enum `RotationDirection`.